sa/sa.go

```diff
@@ -496,40 +494,27 @@ func (ssa *SQLStorageAuthority) countCertificates(domain string, earliest, lates
 		return 0, err
 	}
 
-	// If the `AllowRenewalFirstRL` feature flag is enabled then do the work
-	// required to discount renewals
-	if features.Enabled(features.AllowRenewalFirstRL) {
-		// If there are no serials found, short circuit since there isn't subsequent
-		// work to do
-		if len(serials) == 0 {
-			return 0, nil
-		}
-
-		// Find all FQDN Set Hashes with the serials from the issuedNames table that
-		// were visible within our search window
-		fqdnSets, err := ssa.getFQDNSetsBySerials(serials)
-		if err != nil {
-			return 0, err
-		}
+	// If there are no serials found, short circuit since there isn't subsequent
```

Comment:
The AllowRenewalFirstRL isn't actually enabled in prod due to performance issues with accounts that have a high number of setHashes. We should not delete this feature flag yet.

Reply:
Huh, I thought I saw this in the production config, must of been looking at something else.